packages/vhd-lib/src/vhd.js

```diff
@@ -232,24 +230,14 @@ export default class Vhd {
   // Write functions.
   // =================================================================
 
-  // Write a buffer/stream at a given position in a vhd file.
+  // Write a buffer at a given position in a vhd file.
   async _write(data, offset) {
```

Comment:
We should add `assert(Buffer.isBuffer(data))` to make sure the change is complete.